src/bucket/BucketInputIterator.cpp

```diff
@@ -69,7 +69,8 @@ BucketInputIterator::operator bool() const
     return mEntryPtr != nullptr;
 }
 
-BucketEntry const& BucketInputIterator::operator*()
+BucketEntry const&
```

Comment:
this is noise from not running the right version of `clang-format`

Reply:
I got this after applying the 5.0 version.
I will check and clean it up.